javascript-forms/form-values.js

```diff
@@ -0,0 +1,10 @@
+var $sendButton = document.querySelector('#contact-form');
+
+$sendButton.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var obj = {};
+  obj.name = document.forms[0].name.value;
+  obj.email = document.forms[0].email.value;
+  obj.message = document.forms[0].message.value;
+  console.log(obj);
```

Comment:
You need to reset the form here

Reply:
👍 My bad I thought it was asking to reset in the console